ynput/OpenPype
# TVPaint: Small enhancements

## Brief description
Small enhancements in TVPaint integration which did not get to https://github.com/ynput/OpenPype/pull/4471.

## Description
It was found out that `opacity` returned from `tv_layerinfo` is always empty and is dangerous to add it to layer information. Added information about "current" layer to layers information. Disable review of Render Layer and Render Pass instances by default. In most of productions is used only "scene review". Skip usage of `"enabled"` key from settings in automated layer/pass creation.

## Testing notes:
1. Start TVPaint
2. Open Publisher
3. Render Layer and Render Pass creator should have `Mark for review` set to `False` by default
(Nothing should crash meanwhile)
